.github/workflows/e2e-manual.yaml

```diff
@@ -0,0 +1,82 @@
+name: Manual E2E
+on:
+  workflow_dispatch:
+    inputs:
+      test-suite:
+        description: 'The Test Suite To Run'
+        required: true
+        type: string
+        default: ""
+      chain-a-image:
+        description: 'The image to use for chain A'
+        required: true
+        type: string
+        default: "ghcr.io/cosmos/ibc-go-simd-e2e"
+      chain-a-tag:
+        description: 'The tag to use for chain A'
+        required: true
+        type: string
+        default: "main"
+      chain-b-image:
+        description: 'The image to use for chain B'
+        required: true
+        type: string
+        default: "ghcr.io/cosmos/ibc-go-simd"
+      chain-b-tag:
+        description: 'The tag to use for chain B'
+        required: true
+        default: "v4.0.0-rc2"
+        type: string
+      relayer-tag:
+        description: 'The tag to use for the relayer'
+        required: true
+        default: "v2.0.0-rc2"
+        type: string
```

Comment:
nice! Is the idea to add multiple files, one for each previous version?

Reply:
the way `inputs` work is that you can specify a value when you run the workflow (vi UI or using a cli tool like `gh`). This one workflow should let us run any test suite using any combination of versions.

Reply:
Got it. So we would run this workflow manually rather than it being run on each pr. That makes sense. Mostly to save execution time?

Reply:
yes this is a manual task, we can run the permutations we care about when we are doing QA. I think it might be overkill to run all these permutations on every PR.

Reply:
Sounds good. Could you add instructions on how to run manually? I'd like to test it out myself

Reply:
@colin-axner I added instructions on how to execute these tests with `gh`. Once they are merged into `main`, we will be able to use the UI to trigger the workflows.

Reply:
Adding a sample run here for reference: https://github.com/cosmos/ibc-go/runs/7766177908?check_suite_focus=true